docs/standard_library.md

```diff
@@ -55,4 +72,7 @@ To make sure that API producing error is unstable check
 [`lib.deno.unstable.d.ts`](https://github.com/denoland/deno/blob/master/cli/js/lib.deno.unstable.d.ts)
 declaration.
 
-This problem should be fixed in the near future.
+This problem should be fixed in the near future. Please also note that while the
+std library is currently considered unstable, the `--unstable` flag is only
+required when using the std library if the std module itself uses an unstable
+Deno runtime API.
```

Comment:
It's best not to bring up these two very different types of "unstable" in the same sentence. There is the first meaning that `std` could have breaking changes whenever, and there is the second meaning that parts of `std` use unstable runtime APIs (which is basically a bug which we'll fix and never allow to happen again).

Also, there's no meaningful segmentation of whether or not an unstable API usage shows up in your dependency graph, it's not something users can easily work around. We should word this more flippantly...
```suggestion
This problem should be fixed in the near future. Feel free to omit the flag if the particular modules you depend on compile successfully without it.
```